central/plugins/org.jboss.tools.central/src/org/jboss/tools/central/editors/xpl/DiscoveryViewer.java

```diff
@@ -263,6 +263,7 @@ static Image computeIconImage(AbstractCatalogSource discoverySource, Icon icon,
 					return image;
 				}
 			}
+			return ImageDescriptor.getMissingImageDescriptor().createImage();
```

Comment:
I guess that this is not related to fixing tests, it changes impl behaviour. It now returns "missing image" icon if there's no image path whereas it was returning null before. 
I dont mind though, just trying to understand.

Reply:
yes, it nows return an image as ecxpected by contract, wheras there was some NPE while generating the central page contents.